packages/snaps-execution-environments/src/webview/index.ts

```diff
@@ -0,0 +1,17 @@
+import { BrowserRuntimePostMessageStream } from '@metamask/post-message-stream';
+
+import { executeLockdownEvents } from '../common/lockdown/lockdown-events';
+import { executeLockdownMore } from '../common/lockdown/lockdown-more';
+import { WebviewSnapExecutor } from './WebviewSnapExecutor';
+
+// Lockdown is already applied in LavaMoat
+executeLockdownMore();
+executeLockdownEvents();
+
+// The stream from the RN Webview document to the mobile (caller) execution service.
+const parentStream = new BrowserRuntimePostMessageStream({
```

Comment:
I don't think `BrowserRuntimePostMessageStream` will work here

Reply:
that was outdated.